cspatients/templates/cspatients/patient.html

```diff
@@ -114,22 +114,6 @@ <h4 class="heading heading__subtitle">Delivery Details: Baby {{ baby.baby_number
               <h5 class="patient-list__label">Time of delivery</h5>
               <h4 class="patient-list__entry">{{ baby.delivery_time|time:"H:i" }}</h4>
             </li>
-            <li class="patient-list__item">
-              <h5 class="patient-list__label">Weight of baby</h5>
-              <h4 class="patient-list__entry">{{ baby.baby_weight_grams }}</h4>
-            </li>
-            <li class="patient-list__item">
-              <h5 class="patient-list__label">Apgar 1</h5>
-              <h4 class="patient-list__entry">{{ baby.apgar_1 }}</h4>
-            </li>
-            <li class="patient-list__item">
-              <h5 class="patient-list__label">Apgar5</h5>
-              <h4 class="patient-list__entry">{{ baby.apgar_5 }}</h4>
-            </li>
-            <li class="patient-list__item">
-              <h5 class="patient-list__label">Nicu</h5>
-              <h4 class="patient-list__entry">{{ baby.nicu }}</h4>
```

Comment:
We're removing this in the display, but not making it nullable. This seems contrary from what we're doing on the other fields. Just want to make sure that this is correct.

Reply:
I made provision in the backend incase the values are sent again by the flow but not the frontend, which is probably a better idea.

Let me do that